web/geo/usa-states.geojson

```diff
@@ -1326,9 +1173,7 @@
     },
     {
       "type": "Feature",
-      "properties": {
-        "layer": "USA Borders"
-      },
+      "properties": { "zoneName": "", "stateId": "Unknown" },
```

Comment:
This one should probably be removed :)

Reply:
Good spotting, I've fixed this here #6038 and a few other improvements